packages/model-metrics/src/publishMetrics.ts

```diff
@@ -41,9 +41,9 @@ export class MetricPublisher {
     private modelId: StreamID;
 
     private static readonly NETWORK_MODEL_MAP: Record<Networks, StreamID> = {
-        [Networks.MAINNET] : StreamID.fromString('kjzl6hvfrbw6c7kjl8p225jzy0mjxqo5xdcdt7eqsjv0519arqfda598zfgpfx9'),
-        [Networks.TESTNET_CLAY] : StreamID.fromString('kjzl6hvfrbw6c6z82vn6tfx92nogxf0rukcibdxigity7xe69rptwmz0mb2itdv'),
-        [Networks.DEV_UNSTABLE] : StreamID.fromString('kjzl6hvfrbw6c7kjl8p225jzy0mjxqo5xdcdt7eqsjv0519arqfda598zfgpfx9'),
```

Comment:
hey @stbrody so there is an interesting kind of navel-gazing here where we have to compile the metric and refer to it here; previously i deployed a different model to mainnet, clay and dev-unstable, but actually we *can* deploy the same model to all three. 

Do you have an opinion on whether we should normalize to use different model doc ids on each network, or the same?  @PaulLeCam curious your thoughts as well?  Are models across all networks, or should they be?

Reply:
I think it's good practice to keep the same streamid across networks so it's clear how they map to each other

Reply:
Yes I agree the same model should be used on all networks. If it's a different models, it should have a different name/version/specific identifier.

Reply:
cool @JustinaPetr maybe we want to include taht in the docs somehow, since for me it wasn't an obvious thing, that the model stream id would be the same across networks.  (I guess generally i think of a stream id as tied to a network)